SingularityService/src/test/java/com/hubspot/singularity/data/StateManagerTest.java

```diff
@@ -53,4 +57,21 @@ public void itDoesntCountCleaningTasks() {
     Assert.assertEquals(0, stateManager.getState(true, false).getUnderProvisionedRequests());
     Assert.assertEquals(1, stateManager.getState(true, false).getOverProvisionedRequests());
   }
+
+  @Test
+  public void itDoesntFlagPendingRequestsForUnderOrOverProvisioning() {
+    initRequest();
+    initFirstDeploy();
+
+    SingularityRequest request = requestResource.getRequest(requestId, singularityUser).getRequest();
+
+    requestManager.activate(request.toBuilder().setInstances(Optional.of(0)).build(), RequestHistoryType.UPDATED, System.currentTimeMillis(), Optional.<String> absent(), Optional.<String> absent());
```

Comment:
did you confirm that this test actually fails without the filter you added? Seems odd to have it at instances of `0` (tbh I thought our validation would catch that...)